packages/parser/common.go

```diff
@@ -467,10 +468,11 @@ func (p *Parser) getEGSPrice(name string) (decimal.Decimal, error) {
 }
 
 // CallContract calls the contract functions according to the specified flags
-func (p *Parser) CallContract(flags int) (string, error) {
+func (p *Parser) CallContract(flags int) (ret string, err error) {
```

Comment:
please, use contractResult or something similar instead of ret in return values naming.

Reply:
Fixed.